app/views/devise/registrations/new.html.erb

```diff
@@ -13,12 +13,12 @@
     <p><%= t("authentication.mandatory_warning") %></p>
     <%= f.validation_summary %>
 
-    <div class="registration__row">
-      <div class="registration__field">
+    <div class='registration__row'>
```

Comment:
The standard is double quotes for html attributes https://github.com/moneyadviceservice/frontend-code-standards#html-markup